website/src/components/CaseStudyJumbotron.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+
+const COLORS = {
+    dark: 'bg-black text-light',
+    purple: 'bg-purple text-light',
+}
+
+export const CaseStudyJumbotron: React.FunctionComponent<{
+    title: string
+    titleClassName?: string
+    quote: string
+    author: string
+    url: string
+    image: string
+    buttonText?: string
+    color?: keyof typeof COLORS
+    className?: string
+}> = ({
+    title,
+    titleClassName = 'display-6',
+    quote,
+    author,
+    url,
+    image,
+    buttonText = 'Learn their strategy',
+    color = 'dark',
+    className = '',
+}) => (
+    <div className={`jumbotron case-studies-feature pt-5 pb-3 ${COLORS[color]} ${className}`}>
+        <div className="row container justify-content-center mb-4">
+            <h2 className={`text-center ${titleClassName}`}>{title}</h2>
+        </div>
+        <div className="row container justify-content-center">
+            <div class="col-sm-12 col-md-2 case-studies-feature__image">
+                <a href={url}>
+                    <img class="p-3 img-thumbnail align-middle" src={image} />
+                </a>
+            </div>
+            <div class="col-md-6 case-studies-feature__content">
+                <blockquote class="blockquote">
+                    <p class="mb-1">{quote}</p>
+                    <footer class="blockquote-footer">
+                        <cite title="Thorn Software Engineer Jacob Gilesspie">{author}</cite>
```

Comment:
Should the `title` attribute should be set to `author` too?

Reply:
Fixed - Thanks!